app/viewmodels/journeys/taskList/PrepopTradeJourneyStatusesViewModel.scala

```diff
@@ -0,0 +1,92 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.journeys.taskList
+
+import controllers.journeys.income
+import models._
+import models.common.JourneyStatus.CannotStartYet
+import models.common.{BusinessId, JourneyStatus, TaxYear, TradingName, TypeOfBusiness}
+import models.journeys.Journey
+import models.journeys.Journey._
+import models.requests.TradesJourneyStatuses
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.{SummaryList, SummaryListRow, Value}
+import viewmodels.govuk.summarylist._
+import viewmodels.journeys.{SummaryListCYA, determineJourneyStartOrCyaUrl, getJourneyStatus}
+
+case class PrepopTradeJourneyStatusesViewModel(tradingName: TradingName,
+                                               typeOfBusiness: TypeOfBusiness,
+                                               businessId: BusinessId,
+                                               statusList: SummaryList)
+
+object PrepopTradeJourneyStatusesViewModel {
+
+  def buildPrepopSummaryList(tradesJourneyStatuses: TradesJourneyStatuses, taxYear: TaxYear)(implicit messages: Messages): SummaryList = {
+    implicit val impTaxYear: TaxYear                       = taxYear
+    implicit val businessId: BusinessId                    = tradesJourneyStatuses.businessId
+    implicit val impJourneyStatuses: TradesJourneyStatuses = tradesJourneyStatuses
+
+    val incomePrepopRow            = buildRow(IncomePrepop, dependentJourneyIsFinishedForClickableLink = true)
+    val selfEmploymentPrepopRow    = buildRow(SelfEmploymentPrepop, dependentJourneyIsFinishedForClickableLink = true)
+    val expensesPrepopRow          = buildRow(ExpensesPrepop, dependentJourneyIsFinishedForClickableLink = true)
+    val capitalAllowancesPrepopRow = buildRow(CapitalAllowancesPrepop, dependentJourneyIsFinishedForClickableLink = true)
+    val adjustmentsPrepopRow       = buildRow(AdjustmentsPrepop, dependentJourneyIsFinishedForClickableLink = true)
+
+    val rows: List[SummaryListRow] =
+      List(selfEmploymentPrepopRow, incomePrepopRow, expensesPrepopRow, capitalAllowancesPrepopRow, adjustmentsPrepopRow)
+
+    SummaryListCYA.summaryList(rows)
+  }
+
+  private def buildRow(journey: Journey, dependentJourneyIsFinishedForClickableLink: Boolean)(implicit
+      messages: Messages,
+      taxYear: TaxYear,
+      businessId: BusinessId,
+      journeyStatuses: TradesJourneyStatuses): SummaryListRow = {
+    val status: JourneyStatus = getJourneyStatus(journey, dependentJourneyIsFinishedForClickableLink)
```

Comment:
If dependentJourneyIsFinishedForClickableLink is always true then can remove as a parameter, delete from lines 44-48 and add it as 'true' in line 61